admin/admin-service/src/main/java/io/mosip/admin/service/impl/ApplicantDetailServiceImpl.java

```diff
@@ -114,6 +120,36 @@ public ApplicantDetailsDto getApplicantDetails(String rid) throws Exception {
         }
         return applicantDetailsDto;
     }
+
+    @Override
+    public byte[] getRIDDigitalCard(String rid, boolean isAcknowledged) throws Exception {
+        String data=null;
+        DigitalCardStatusResponseDto digitalCardStatusResponseDto = getDigitialCardStatus(rid);
```

Comment:
Lets get the card status only after the acknowledge check is done

Reply:
taken care

Reply:
Resolved